Android/app/src/main/java/com/example/cardier/recycler/MessageAdapter.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.cardier.recycler
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.cardier.R
+import com.google.firebase.auth.FirebaseAuth
+
+class MessageAdapter(val context: Context, val messageList: ArrayList<com.example.cardier.data.model.Message>):
+    RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+
+    val ITEM_RECEIVED = 1
+    val ITEM_SENT = 2
+
+    class SentViewHolder(itemView: View): RecyclerView.ViewHolder(itemView){
+        val sentMessage = itemView.findViewById<TextView>(R.id.textSentMessage)
+    }
+
+    class ReceiveViewHolder(itemView: View): RecyclerView.ViewHolder(itemView){
+        val receiveMessage = itemView.findViewById<TextView>(R.id.textReceivedMessage)
+    }
+
+    override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
+
+        val currentMessage = messageList[position]
+
+        if (holder.javaClass == com.example.cardier.recycler.MessageAdapter.SentViewHolder::class.java) {
```

Comment:
needs to be `com.github.osucoding.cardier`

Reply:
also, is there any way to make this shorter by having an import statement at the top of the file? Like 
`import com.github.osucoding.cardier.recycler.MessageAdapter`?